pkg/netpol/connlist/conns_formatter_dot.go

```diff
@@ -72,9 +73,11 @@ func (d formatDOT) writeOutput(conns []Peer2PeerConnection) (string, error) {
 		externalPeersLines = categorizeAndAddPeerLine(conns[index].Src(), peersVisited, externalPeersLines, nsPeers)
 		externalPeersLines = categorizeAndAddPeerLine(conns[index].Dst(), peersVisited, externalPeersLines, nsPeers)
 	}
-	for _, val := range d.peersList {
-		if !val.IsPeerIPType() {
-			externalPeersLines = categorizeAndAddPeerLine(val, peersVisited, externalPeersLines, nsPeers)
+	if d.focusWorkload == "" {
```

Comment:
what if focus workload is given, and the required workload is not connected at all?

Reply:
Right, handled it